### ISSUE TITLE
Create New Folder Button for Caja

### ISSUE THREAD

**Issue by MalwaredPC** [mate-desktop/caja#677]
I wish there were a button for creating new folders in the actual folder in Caja's toolbar. Thank you.

**Comment by kn00tcn** [comment]
what about the rest of common user actions? rename, cut copy paste, undo

**Comment by MalwaredPC** [comment]
Yes, definitively. There's enough space to put some buttons for common actions there. I'm not fan of undo in my case but that' something common.

**Comment by UbuntuMetal** [comment]
Hi. While most would agree that there should be a user-friendly way to customise the toolbar - and hopefully that option gets added in the near future - you can in fact add a **[Create New Folder button to Caja's toolbar](https://ubuntugenius.wordpress.com/2022/01/13/caja-toolbar-add-create-new-folder-button-to-ubuntu-mates-file-manager/)**, so click the link for how to do this. Also, **[you can add 20 toolbar buttons](https://ubuntugenius.wordpress.com/2022/02/03/caja-toolbar-full-list-of-buttons-you-can-add-to-ubuntu-mates-file-manager/)** in total, so click that link for the index page, where you can browse for the buttons you'd like to add to your custom toolbar. Have fun!

**Comment by lukefromdc** [comment]
Note that that will stop working if the UI files im question are ever converted to gresources (which load as part of thr binary for speed).

That might be a reason not to do so if this is a common point of user customization. The code used in EOM and some of the other MATE apps would be the obvious approach to allowing toobar customization from the application itself.
That in turn would remove this comcern when comverting the ui files to gresources.
